tests/rt.conf

```diff
@@ -1,3 +1,8 @@
+
+
+# FIXME: CHANGES TO THIS FILE SHOULD NOT BE MERGED TO DEVELOP
```

Comment:
Should this file be returned to the develop version?

Reply:
Yes.